openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -129,6 +129,7 @@
  */
 @Slf4j
 public abstract class EntityRepository<T extends EntityInterface> {
+  private static final String EXTENSION = "extension";
```

Comment:
Instead of creating a new static variable we could use `FIELD_EXTENSION` from `Entity.java`, wdyt?